lib/web/app/middleware.go

```diff
@@ -64,6 +64,12 @@ func (h *Handler) withAuth(handler handlerAuthFunc) http.HandlerFunc {
 // redirectToLauncher redirects to the proxy web's app launcher if the public
 // address of the proxy is set.
 func (h *Handler) redirectToLauncher(w http.ResponseWriter, r *http.Request, p launcherURLParams) error {
+	// When handling requests with certificate, redirecting to launcher won't
```

Comment:
Could you extend the command and explain when this scenario happens ?

Reply:
I've changed the comment and the redirect rule to clarify it.